go.mod

```diff
@@ -2,16 +2,16 @@
 //
 // SPDX-License-Identifier: CC0-1.0
 
-module github.com/ssb-ngi-pointer/go-metafeed
+module github.com/ssbc/go-metafeed
 
 go 1.16
 
 require (
 	github.com/kr/pretty v0.1.0 // indirect
+	github.com/ssbc/go-ssb-refs v0.5.2-0.20221017100922-8e95413c6580
```

Comment:
Refs latest commit of https://github.com/ssbc/go-ssb-refs/pull/2 :+1: 

Probably we should do a proper release at some point but this works For Now :tm: